docs/general/celo_l1_index.md

```diff
@@ -1,3 +1,5 @@
-# Overview 
+# Overview
 
-Celo L1 is a mobile-friendly blockchain focused on financial inclusion. It enables fast, low-cost transactions, supports stablecoins like cUSD, and uses phone numbers for payments. Built on proof-of-stake, it’s carbon-neutral and powers a growing DeFi ecosystem.
+Until March 26, 2025 at block height 31,056,500, Celo L1 has been a mobile-friendly blockchain focused on financial inclusion. It enables fast, low-cost transactions, supports stablecoins like cUSD, and uses phone numbers for payments. Built on proof-of-stake, it’s carbon-neutral and powers a growing DeFi ecosystem.
+
+The journey continues with the [migration to L2](/cel2)!
```

Comment:
something about this reads to me like its a completely new / different project. 

Maybe because the wording "the journey continues" is so often used by projects/ companies when they go kaput.